apps/settings/js/bluetooth.js

```diff
@@ -669,16 +682,18 @@ navigator.mozL10n.once(function bluetoothSettings() {
 
     function setDeviceDisconnect(device, callback) {
       if (!bluetooth.enabled || !defaultAdapter ||
-          device.address !== connectedAddress) {
-        if (callback)
-          callback();
-        return;
+        device.address !== connectedAddress) {
```

Comment:
Is the indent expected here? In my view point, two indents is more readable for me. I'm not sure which style is fit rule.